config/core/test-configs.yaml

```diff
@@ -1927,6 +1932,7 @@ test_configs:
       - baseline
       - baseline-nfs
       - cros-ec
+      - iec-security
```

Comment:
Maybe drop `hp-x360-12b-n4000-octopus` as it's also enabled on `asus-C523NA-A20057-coral` and they are both x86_64 Intel Chromebook devices (not quite same SoC / CPU generation, but probably not worth running twice for this type of high-level security feature test).

Reply:
dropped